src/bin/nydus-image/main.rs

```diff
@@ -348,6 +350,56 @@ fn prepare_cmd_args(bti_string: &'static str) -> App {
                 )
         );
 
+    let app = app.subcommand(
+            App::new("chunkdict")
+                .about("deduplicate RAFS filesystem metadata")
+                .subcommand(
+                    App::new("save")
+                        .about("Save chunk info to a database")
+                        .arg(
+                            Arg::new("bootstrap")
+                            .short('B')
+                            .long("bootstrap")
+                            .help("File path of RAFS meta blob/bootstrap")
+                            .conflicts_with("BOOTSTRAP")
+                            .required(false),
+                        )
+                    .arg(
+                        Arg::new("database-path")
```

Comment:
How about using a database URL like `sqlite:///path/to/image.db`?

Reply:
Ok.